test/parallel/test-module-loading-error.js

```diff
@@ -28,7 +28,8 @@ const errorMessagesByPlatform = {
   win32: ['%1 is not a valid Win32 application'],
   linux: ['file too short', 'Exec format error'],
   sunos: ['unknown file type', 'not an ELF file'],
-  darwin: ['file too short']
+  darwin: ['file too short'],
+  aix: ['Cannot load module', 'Cannot run a file that does not have a valid format.']
```

Comment:
This line exceeds 80 characters, please format the code according to our style guidelines.